functions/concepts/src/main/java/com/example/functions/concepts/LazyFields.java

```diff
@@ -36,18 +36,21 @@
   // Uses the "initialization-on-demand holder" idiom
   // More information: https://en.wikipedia.org/wiki/Initialization-on-demand_holder_idiom
   private static class LazyGlobalHolder {
+    // Making the default constructor private prohibits instantiation of this class
```

Comment:
**N.B:** wrong commit name, should be `[comments] use proper IDOH pattern`

Reply:
@ace-n  I supper appreciate you taking the time to comment these well. These are great.